src/core/ext/lb_policy/grpclb/grpclb.c

```diff
@@ -155,9 +158,8 @@ typedef struct wrapped_rr_closure_arg {
   /* The RR instance related to the closure */
   grpc_lb_policy *rr_policy;
```

Comment:
The reasoning behind having the `rr_policy` pointer instead of a `glb_policy` pointer in the closure arg was to limit how much was exposed to the arg.